src/main/java/name/valery1707/kaitai/KaitaiException.java

```diff
@@ -1,6 +1,8 @@
 package name.valery1707.kaitai;
 
+@SuppressWarnings("WeakerAccess")
 public class KaitaiException extends Exception {
+	@SuppressWarnings("unused")
```

Comment:
\[Checkstyle\] WARNING: 'ctor def modifier' has incorrect indentation level 8, expected level should be 2\.